src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
Hi David (@dglazer),

Thank you for the thoughtful and insightful comments, but with the highest esteem and most respect, why are we creating limits based on a design decision at Google?  I read most of the Google papers - and many others - and I know what is possible both from them and the different subject areas in computer science.

I posted the following for a reason:

https://github.com/ga4gh/schemas/issues/131

For instance from the Mesa paper - which one can read from [the following link](http://research.google.com/pubs/archive/42851.pdf) - there is a clear line that says:

_We show how schema changes for a large number of tables can be performed dynamically and efficiently without affecting correctness or performance of existing applications._

Data and schemas are two separate things.  I'm not saying let's use Mesa - which would be nice - but let's implement concepts that utilize some of that functionality in a public setting.  If you're up for it, let's re-engineer Google's backend for the GA4GH implementation to make it flexible enough with future implementations.  It is just a service, and that means the code does not have to be public - which currently is not, and thus satisfies that requirement as well.

I have worked at too many places to see unnecessary limits set because of an earlier design decision.  We have an opportunity here to do things right.  Let's put our minds together and make it right.

Let me know what you think.

Thanks,
Paul